scripts/send_export_jobs.sh

```diff
@@ -23,8 +23,8 @@ for SNAPSHOT in ${SNAPSHOT_LIST}; do
         echo "Scheduling ${SNAPSHOT_NAME} covering from ${SNAPSHOT_EPOCH_FROM} to ${SNAPSHOT_EPOCH_TO}"
     else
         # Pad the epoch with 0s to 10 digits
-        PADDED_SNAPSHOT_EPOCH_FROM=$(printf "%010d" "$SNAPSHOT_EPOCH_FROM")
-        PADDED_SNAPSHOT_EPOCH_TO=$(printf "%010d" "$SNAPSHOT_EPOCH_TO")
-        envsubst < gce_batch_job.json | gcloud --billing-project protocol-labs-data beta batch jobs submit lily-backfill-snapshot-"$PADDED_SNAPSHOT_EPOCH_FROM"-"$PADDED_SNAPSHOT_EPOCH_TO"-"$(date +%s)" --location europe-north1 --config=-
+        PADDED_SNAPSHOT_EPOCH_FROM=$(printf "%07d" "$SNAPSHOT_EPOCH_FROM")
+        PADDED_SNAPSHOT_EPOCH_TO=$(printf "%07d" "$SNAPSHOT_EPOCH_TO")
+        envsubst < gce_batch_job.json | gcloud --billing-project protocol-labs-data beta batch jobs submit lily-snapshot-"$PADDED_SNAPSHOT_EPOCH_FROM"-"$PADDED_SNAPSHOT_EPOCH_TO"-"$(date +%s)" --location us-central1 --config=-
```

Comment:
Moving to `us-central1` to be where the GCS is and making the padding shorted so its easier to read!